LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/authentication/LabApiAuthenticationClient.java

```diff
@@ -32,42 +38,59 @@
  * A an authentication client that can acquire access tokens for the Microsoft Identity Lab Api.
  */
 public class LabApiAuthenticationClient implements IAccessTokenSupplier {
-
     private final static String SCOPE = "https://msidlab.com/.default";
-
     private final static String TENANT_ID = "72f988bf-86f1-41af-91ab-2d7cd011db47";
-
     private final static String AUTHORITY = "https://login.microsoftonline.com/" + TENANT_ID;
-
-    private final IConfidentialAuthClient mConfidentialAuthClient;
     private final static String CLIENT_ID = "f62c5ae3-bf3a-4af5-afa8-a68b800396e9";
-    private final String mClientSecret;
-
-
-    public LabApiAuthenticationClient(@NonNull final IConfidentialAuthClient confidentialAuthClient,
-                                      @NonNull final String clientSecret) {
-        mConfidentialAuthClient = confidentialAuthClient;
-        mClientSecret = clientSecret;
-    }
+    private final static String KEYSTORE_TYPE = "Windows-MY";
+    private final static String KEYSTORE_PROVIDER = "SunMSCAPI";
+    private final static String CERTIFICATE_ALIAS = "LabVaultAccessCert";
+    private final String mLabSecret;
 
-    public LabApiAuthenticationClient(@NonNull final String clientSecret) {
-        this(ConfidentialAuthClientFactory.INSTANCE.getConfidentialAuthClient(), clientSecret);
+    public LabApiAuthenticationClient(@NonNull final String labSecret) {
+        mLabSecret = labSecret;
```

Comment:
I believe labSecret here, is a path to the Certificate file, correct? I think we should update the variable to make it more clear.

Reply:
It can be the secret or the cert file.